inc/ocf_def.h

```diff
@@ -176,6 +176,8 @@ typedef enum {
 } ocf_cache_mode_t;
 
 #define OCF_SEQ_CUTOFF_MAX_STREAMS 256
+#define OCF_SEQ_CUTOFF_MIN_THRESHOLD 1
+#define OCF_SEQ_CUTOFF_MAX_THRESHOLD 4194181
```

Comment:
why not UIN32_T?

Reply:
I've seen this value in requirements: https://github.com/Open-CAS/open-cas-linux/pull/699/files